layouts/default.vue

```diff
@@ -1,146 +1,147 @@
 <template>
-  <ClientOnly>
-    <v-app
-      class="l-content_wrap"
-      :class="{ 'p-dashboard': isLoggedIn, 'p-login': !isLoggedIn }"
+  <v-app
+    class="l-content_wrap"
+    :class="[
+      authUser.member_id ? 'p-dashboard' : 'p-login',
```

Comment:
Nuxt emits warning msg if window refresh with loggedIn,
whereas does not emit error if user is not loggedIn.

This is because SSR never apply `authUsre.member_id` true due to Cookie login, which runs on only client side.
SSR renders as 'p-login' is user once loggedIn and Client renders as 'p-dashboard' because Cookie keeps loggedIn.
This causes Hydration mismatch.

So this warning is not a problem unless we use SSG.

https://diverta.gyazo.com/d5fbb398fd105b41c73c35f6bf94818b

Reply:
Reverted to your commit https://github.com/diverta/front_nuxt_auth/pull/41/files
Now this PR is in line with `nuxt3`

@yabe-diverta san please confirm